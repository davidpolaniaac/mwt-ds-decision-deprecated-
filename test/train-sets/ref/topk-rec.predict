1.209452  J-A 
2.217164  J-M 

1.560888  M-J 
1.673134  M-A 

1.424552  A-J 
1.644751  A-M 

