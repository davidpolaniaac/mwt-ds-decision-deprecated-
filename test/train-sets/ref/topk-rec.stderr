Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = topk.predict
using no cache
Reading datafile = train-sets/topk.vw
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.000000   0.000000            3         3.0   2.0000   2.0000        4
0.000000   0.000000            6         6.0   3.0000   2.9992        4
0.000000   0.000000           11        11.0   3.0000   2.9999        4

finished run
number of examples per pass = 12
passes used = 1
weighted example sum = 12
weighted label sum = 0
average loss = 1.09109e-07
best constant = 0
total feature number = 39
